app/models/conference_preference.rb

```diff
@@ -1,5 +1,21 @@
 class ConferencePreference < ActiveRecord::Base
+  validates :terms_accepted?, acceptance: true
```

Comment:
The problem I see with this validation is that it will always require the terms to be accepted with or without selected conferences. This forces the user to accept the terms every time he 'edit' his team.